packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -65,6 +65,24 @@ Class name added to the placeholder.
 - Type: `String`
 - Required: No
 
+### disableDropZone
```

Comment:
Hi @desaiuditd, would it be possible to share more information about the use case you have that needs a media placeholder without the option for drag & drop? What is the main reason for wanting the disable drag & drop in that use case?

Reply:
@jorgefilipecosta 

I'm a colleague of Udit's so can answer your question about the use case.

We have a custom media frame implementation that has required fields for the user to fill in. The drop zone component allows them to work around the required fields.

Eventually we're hoping to replace or extend the drop zone to open the media library with the uploaded image selected and ready to be inserted once the required fields are filled in. But this is a little way down the track.

There's further discussion in the original pull request, #17053.

Reply:
After Pete’s reply, my 2 cents which I had written in the PR description:

> I work for a newsroom where we do not allow our editors/reporters to drag & drop images into the post, because we have our own media service to handle the images.